lib/Service/AttachmentService.php

```diff
@@ -137,16 +137,17 @@ public function findAll($cardId, $withDeleted = false) {
 
 	/**
 	 * @param $cardId
+	 * @param bool $update | Force the update of the cached values 
 	 * @return int|mixed
 	 * @throws BadRequestException
 	 */
-	public function count($cardId) {
+	public function count($cardId, $update = false) {
 		if (is_numeric($cardId) === false) {
 			throw new BadRequestException('card id must be a number');
 		}
 
 		$count = $this->cache->get('card-' . $cardId);
-		if (!$count) {
+		if ($update OR !$count) {
```

Comment:
```suggestion
		if ($update || !$count) {
```

Just to stick to our code style ;)